### PR TITLE
Separate mothership and component reconciler binaries

### PR DESCRIPTION
**Pull request by suleymanakbas91**
See #190

### CODE REVIEW

**Review by tobiscr — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -20,18 +20,18 @@ Follow these steps to run Reconciler locally:
...
    ```
 
    > **CAUTION:** At the moment, reconciliation with default values will fail. You must specify additional parameters:
    > ```bash
-   > ./bin/reconciler-darwin local global.ingress.domainName=example.com,global.domainName=example.com,tracing.authProxy.config.useDex=false,tracing.virtualservice.enabled=false,kiali.authProxy.config.useDex=false,kiali.virtualservice.enabled=false
+   > ./bin/mothership-darwin local global.ingress.domainName=example.com,global.domainName=example.com,tracing.authProxy.config.useDex=false,tracing.virtualservice.enabled=false,kiali.authProxy.config.useDex=false,kiali.virtualservice.enabled=false
    > ```
 
 
    Alternatively, you can run Reconciler with the specified components:
```
Comment (NHingerl) [inline README.md:32]: I suggest to add indentation to this paragraph so that the alternative is also part of the Caution note.

**Review by NHingerl — APPROVED** [review]

**Pull request merged**